tests/runtime/probe

```diff
@@ -100,6 +100,17 @@ WILL_FAIL
 NAME kprobe_offset_module
 RUN {{BPFTRACE}} -e 'kprobe:nft_trans_alloc_gfp+0x5 { printf("hit\n"); exit(); }' -c '/usr/sbin/nft add table bpftrace'
 EXPECT hit
+ARCH x86_64|aarch64
+TIMEOUT 5
+REQUIRES lsmod | grep '^nf_tables'
+REQUIRES /usr/sbin/nft --help
+AFTER nft delete table bpftrace
```

Comment:
I think `AFTER` is wrong here. See: https://github.com/iovisor/bpftrace/blob/31d7a35816fa4993ae2f1208153ce38c632be567/tests/runtime/engine/runner.py#L245

It's run right after probe attachment. I think a `CLEANUP` directive is needed

Reply:
CLEANUP directive PR sent.

Reply:
@viktormalik Working on the issue.